channels-rke2.yaml

```diff
@@ -595,6 +595,26 @@ releases:
         version: v3.8-build2021110403
     featureVersions: &featureVersions-v1-22-11-rke2r1
       <<: *featureVersions-v1-21-14-rke2r1
+  - version: v1.22.13-rc1+rke2r1
+    minChannelServerVersion: v2.6.3-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v1-22-11-rke2r1
```

Comment:
Can we rename this to `&serverArgs-v1-22-13-rke2r1` OR maybe it's finally time to make this easy like k3s and rename it to: `&serverArgsv122-1` with the intention of next release being `&serverArgsv122-2` if there are any changes, and if there aren't then just not renaming at all. Similarly for 1.23 and 1.24 we'd have their own.

Reply:
Good catch @rancher-max